### ISSUE TITLE
[Feature] Allow skipping specific browser downloads

### ISSUE THREAD

**Issue by jperl** [microsoft/playwright#823]
My library depends on `playwright`, and supports all the browsers playwright supports. However not all of my users want to download each browser. 

It would be nice if the end-user of my library could skip certain browser downloads by setting environment variables.

For example, if they only care about chromium they could do `PLAYWRIGHT_SKIP_FIREFOX_DOWNLOAD=true PLAYWRIGHT_SKIP_WEBKIT_DOWNLOAD=true npm install`

**Comment by pavelfeldman** [comment]
Closing as a dupe of https://github.com/microsoft/playwright/issues/812

**Comment by jperl** [comment]
@pavelfeldman This is not a dupe of #812. `qawolf` depends on `playwright`. It would be a lot of maintenance burden to create `qawolf-firefox`, `qawolf-webkit`, `qawolf-chromium` packages. It would be much nicer to just allow ENV variables to skip the download.

**Comment by jperl** [comment]
This added to [downloadBrowser](https://github.com/microsoft/playwright/blob/master/download-browser.js) would save a lot of effort 

```js
if (process.env[`PLAYWRIGHT_SKIP_${browserType.toUpperCase()}_DOWNLOAD`] === "true"] {
  return;
}
```

**Comment by aslushnikov** [comment]
@jperl Yeah I agree, this is different.

I don't like using ENV variables as an API for libraries. We tried this in Puppeteer with [`PUPPETEER_SKIP_CHROMIUM_DOWNLOAD`](https://pptr.dev/#?product=Puppeteer&version=v2.1.0&show=api-environment-variables), and it bit us: if different dependencies used Puppeteer somewhere deep in their guts without users even knowing it, and users tried to use Puppeteer and disable browser downloads, then this env variable got propagated during `npm install` step, breaking dependencies in a weird ways.

So instead we switched to programmatic browser downloads: every library that wished to use Puppeteer internally had to use `puppeteer-core` instead, and have it's own simple `install.js` script that uses [`BrowserFetcher API`](https://pptr.dev/#?product=Puppeteer&version=v2.1.0&show=api-class-browserfetcher) to download browser, if necessary. This way dependencies stay hermetic. (more info: [puppeteer vs puppeteer-core](https://pptr.dev/#?product=Puppeteer&version=v2.1.0&show=api-puppeteer-vs-puppeteer-core)).

---

Now, `qawolf` wants to use playwright internally and is supposed to be an end-user product rather then a library. This means that `qawolf` should have it's own mechanism to specify which browsers to download, and use `playwright-core` internally to drive and download browsers.

It looks like we don't currently expose browser downloads - let me fix it.
